packages/jest-circus/src/__mocks__/testUtils.ts

```diff
@@ -46,6 +46,7 @@ export const runTest = (source: string) => {
 
     const testEventHandler = require('${TEST_EVENT_HANDLER_PATH}').default;
     const addEventHandler = require('${CIRCUS_STATE_PATH}').addEventHandler;
+    const removeEventHandler = require('${CIRCUS_STATE_PATH}').removeEventHandler;
```

Comment:
unfortunately, the test it self doesn't cover the multi imports issue. However, at least it covers desired behavior of `addEventHandler` and `removeEventHandler`.